source/collections/HashDictionary.ooc

```diff
@@ -3,15 +3,30 @@ import structs/[HashBag, HashMap, ArrayList]
 HashDictionary: class {
 	_myHashBag: HashBag
 	_capacity: Int
+	size ::= this _myHashBag size()
```

Comment:
Would it be a good idea to call this `count` instead, as in our other collections?